### PR TITLE
Issue 1055 gen keys via pgpainless

### PR DESCRIPTION
**Pull request by DenBond7**

### CODE REVIEW

File: FlowCrypt/src/main/java/com/flowcrypt/email/extensions/KeyRingInfoExt.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * If the ring is a [PGPPublicKeyRing], return false.
+ *
+ * @return true if all secret keys are encrypted.
+ */
+fun KeyRingInfo.isFullyEncrypted(): Boolean {
+  if (isSecretKey) {
+    for (secretKey in secretKeys) {
+      if (secretKey.keyEncryptionAlgorithm == SymmetricKeyAlgorithm.NULL.algorithmId) {
+        return false
+      }
```
Comment (tomholub) [inline FlowCrypt/src/main/java/com/flowcrypt/email/extensions/KeyRingInfoExt.kt:30]: Need to cross check this `isFullyEncrypted` - for one thing there is a dummy gnu S2K requiring special treatment, second there could be some null-like encryption method like plain. (sometimes the spec is messy). That will be for another PR.

**Review by tomholub — APPROVED** [review]

File: FlowCrypt/src/main/java/com/flowcrypt/email/extensions/KeyRingInfoExt.kt (hunk 1 above)
Comment (DenBond7) [reply FlowCrypt/src/main/java/com/flowcrypt/email/extensions/KeyRingInfoExt.kt:30]: ref https://github.com/pgpainless/pgpainless/issues/99
